spec/penalty-payment-api.yaml

```
@@ -35,6 +35,23 @@ paths:
             'application/json':
               schema:
                 $ref: '#/components/schemas/ServiceUnavailable'
+  /penalty-payment-api/penalty-reference-types:
+    get:
+      tags:
+        - PenaltyReferenceTypes
```

I think it looks / reads better with spaces here?

```suggestion
        - Penalty Reference Types
```